### ISSUE TITLE
makeRequest and dry run

### ISSUE THREAD

**Issue by florian-milky** [contentful/contentful-migration#648]
## Expected Behavior

makeRequest is executed when the migration has been validated and confirmed

## Actual Behavior

makeRequest runs when the migration is validated

## Steps to Reproduce

1. write a migration
2. use makeRequest
3. run the migration
4. observe that makeRequest ran even though you still haven't chosen to run the migration in the cli

## Environment

-   **Node Version**: 
-   **Package Manager Version**: 
-   **Operating System**: 
-   **Package Version**:

**Comment by nteague22** [comment]
Yes, I even have looked at using the context and wiring up getClient, but having the migrations seed pass some form of flag even to allow for developer conditional execution would be helpful -- i.e. pass if it is committing the change or validating, so it is safe to run / does not have side effects
